app/controllers/events_controller.rb

```diff
@@ -5,6 +5,8 @@ class EventsController < ApplicationController
   before_action :authorize_event_author, only: [:update, :destroy]
 
   def index
+    @events = @project.events
+    @project_id = @project.id
```

Comment:
in your view you have @project object available and you can do things like this:
link_to @project 
or project_path(@project)